app/models/author.rb

```diff
@@ -0,0 +1,7 @@
+class Author < ActiveRecord::Base
+  has_many :books
+
+  def last_5_books(author)
+    author.books.order(published_at: :desc).first 5
```

Comment:
@Alexneisc you don't need author parameter here.
```ruby
def last_5_books
  books.order...
end
```
And please move `order(...).first(5)` to `Book` models scope like: `recent_five`